packages/client/src/modules/common/components/web/ui-bootstrap/components/Table.jsx

```diff
@@ -24,20 +28,146 @@ const renderData = (columns, entry) => {
   });
 };
 
-const Table = ({ dataSource, columns, ...props }) => {
+class TablePagination extends React.Component {
```

Comment:
I think there should be both `Table` and `TablePagination` UI components

Reply:
Ok, got you, will move TablePagination component to a separate file.

Reply:
Done in https://github.com/sysgears/apollo-universal-starter-kit/pull/713/commits/b06e7f78abaadd8603d6f4b5b2c518674b642e93